medicines/web/components/search-results/index.tsx

```diff
@@ -221,7 +221,7 @@ const SearchResults = (props: {
           )}
           <p className="ema-message">
             If the product information you are seeking does not appear below, it
-            is possible that the product holds a central European licence and
+            is possible that the product holds an European licence and
```

Comment:
Is it **an** European?

Reply:
i looked it up and consensus says 'a' so i'll change it back 

http://www.bbc.co.uk/worldservice/learningenglish/radio/specials/1521_gramchallenge27/